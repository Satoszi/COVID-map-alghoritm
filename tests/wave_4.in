20 20 2 128
wave_4
++..++..++..++..++..
||++||++||++||++||++
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
||||||||||||||||||||
|||++||++||++||++|||
|++..++..++..++..++|