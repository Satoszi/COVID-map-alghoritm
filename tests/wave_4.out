19 19
13 19
10 19
1 19
0 19
18 18
17 18
11 18
8 18
6 18
5 18
16 17
15 17
14 17
9 17
4 17
3 17
2 17
19 16
13 16
12 16
10 16
7 16
1 16
0 16
18 15
17 15
11 15
8 15
6 15
5 15
16 14
15 14
14 14
9 14
4 14
3 14
2 14
19 13
13 13
12 13
10 13
7 13
1 13
0 13
18 12
17 12
11 12
8 12
6 12
5 12
16 11
15 11
14 11
9 11
4 11
3 11
2 11
19 10
13 10
12 10
10 10
7 10
1 10
0 10
18 9
17 9
11 9
8 9
6 9
5 9
16 8
15 8
14 8
9 8
4 8
3 8
2 8
19 7
13 7
12 7
10 7
7 7
1 7
0 7
18 6
17 6
11 6
8 6
6 6
5 6
16 5
15 5
14 5
9 5
4 5
3 5
2 5
19 4
13 4
12 4
10 4
7 4
1 4
0 4
18 3
17 3
11 3
8 3
6 3
5 3
16 2
15 2
14 2
9 2
4 2
3 2
2 2
19 1
13 1
12 1
10 1
7 1
1 1
0 1
17 0
8 0
5 0
